src/python/antares_xpansion/input_parser.py

```diff
@@ -38,7 +38,7 @@ def _initialize_parser(self):
             "--method",
             dest=LauncherOptionsKeys.method_key(),
             type=str,
-            choices=["benders", "mergeMPS", "adequacy_criterion"],
+            choices=["benders", "mergeMPS", "presolve", "adequacy_criterion"],
```

Comment:
I am not sure that `presolve` has the same status as other choices in this list. For me, it looks more like a step (such as `problem_generation`), that could be optional, called only if solver Xpress is chosen, and that should appear (as an optional step) in the full run. Any remark on these reflexions are welcome @JasonMarechal25 @ianmnz 
Moreover, if included in the full run workflow, there will be a work to handle outputs of the simulation (as operational costs would be those of the presolved pb, not the original one. Candidates should be ok, and that is the most important part).

Reply:
It's true that, given the current setting, presolve is seen more as an solver option itself than a step before the solver. We would need to think on how to integrate in a full run, for instance